imports/startup/server/index.coffee

```diff
@@ -156,6 +156,36 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date, date2) ->
+    query = prefixes + """
+      SELECT ?resolvedTerm
+          (sample(?termLabel) as ?word)
+          (count(DISTINCT ?article) as ?count)
+		      (count(DISTINCT ?article2) as ?count2)
+          ((xsd:float(?count2)/xsd:float(?count)*xsd:float(100)) AS ?result)
+      WHERE {
+        ?phrase anno:category "diseases"
+        ; ^dc:relation ?resolvedTerm
+        ; anno:source_doc ?article
+        .
+        ?article pro:date ?dateTime .
+        ?resolvedTerm rdfs:label ?termLabel
+        FILTER (?dateTime > "#{escape(date)}"^^xsd:dateTime)
+
+    		OPTIONAL{
```

Comment:
It would probably be more efficient to use separate queries than an optional clause that finds every combination of mentions.